src/measure_theory/integration.lean

```diff
@@ -172,6 +172,10 @@ coe_injective $ by simp
 @[simp] lemma piecewise_empty (f g : α →ₛ β) : piecewise ∅ measurable_set.empty f g = g :=
 coe_injective $ by simp
 
+lemma support_indicator [has_zero β] {s : set α} (hs : measurable_set s) (f : α →ₛ β) :
+  function.support (f.piecewise s hs (simple_func.const α 0)) = s ∩ function.support f :=
```

Comment:
here and in `measure_lt_top_of_mem_ℒp_indicator`, I find strange to have indicator in the name but piecewise in the statement. I understand that these will be used by people wanting to prove things about indicators, so having indicator in the name is perhaps indeed the best solution to help those people find the lemmas. I don't know.

Reply:
I agree, I was struggling with the naming.  There is also a distinction one might want to draw between the first of these lemmas, which is piecewise of a general `f` with a constant, and the second, which is piecewise of two constants.  I am very happy to switch to any naming idea you have!